### PR TITLE
{devel}[GCCcore/8.3.0] PCRE v8.43

### PR DESCRIPTION
**Pull request by smoors**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by smoors** [comment]
Test report by @smoors
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node350.hydra.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz, Python 2.7.5
See https://gist.github.com/715f733c35216458264212f77707db5e for a full test report.

File: easybuild/easyconfigs/p/PCRE/PCRE-8.43-GCCcore-8.3.0.eb [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+]
+sources = [SOURCELOWER_TAR_GZ]
+checksums = ['0b8e7465dc5e98c757cc3650a20a7843ee4c3edf50aaf60bb33fd879690d2c73']
+
+builddependencies = [
+    ('binutils', '2.32'),
+]
+
+dependencies = [
+    ('bzip2', '1.0.6'),
```
Comment (boegel) [inline easybuild/easyconfigs/p/PCRE/PCRE-8.43-GCCcore-8.3.0.eb]: @smoors You should bump this to `1.0.8`...

File: easybuild/easyconfigs/p/PCRE/PCRE-8.43-GCCcore-8.3.0.eb [hunk 2]
```diff
@@ -0,0 +1,45 @@
+easyblock = 'ConfigureMake'
+
+name = 'PCRE'
+version = '8.43'
+
+homepage = 'http://www.pcre.org/'
```
Comment (boegel) [inline easybuild/easyconfigs/p/PCRE/PCRE-8.43-GCCcore-8.3.0.eb]: please update to use `https://`

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node3122.skitty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 3.6.8
See https://gist.github.com/2852624f8fd5e164f394d58fa9c3c529 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
generoso - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2660 v4 @ 2.00GHz, Python 3.6.8
See https://gist.github.com/0b818c826ca338e09e8cb1941621098d for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2623.swalot.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/c07e5a18f1c556612aa5a974c1dfdc70 for a full test report.

**Comment by boegel** [comment]
Going in, thanks @smoors!

**Pull request merged**